flink-tests/src/test/java/org/apache/flink/test/checkpointing/UnalignedCheckpointRescaleITCase.java

```diff
@@ -459,46 +478,59 @@ public void processBroadcastElement(Long value, Context ctx, Collector<Long> out
         }
     }
 
-    @Parameterized.Parameters(name = "{0} {1} from {2} to {3}, buffersPerChannel = {4}")
+    @Parameterized.Parameters(
+            name = "{0} {1} from {2} to {3}, sourceSleepMs = {4}, buffersPerChannel = {5}")
```

Comment:
I would add a comment above this line explaining why do we want to have non zero `sourceSleepMs` sometimes. That we want to test the rescaling without backpressure with only occasional a couple of captured in-flight records .